src/mozilla_sec_eia/utils/cloud.py

```diff
@@ -134,45 +135,59 @@ def from_path(
             )
 
 
-class GCSArchive(BaseSettings):
-    """Settings for connecting to GCS resources.
+class GoogleCloudSettings(BaseSettings):
+    """Load environment variables to manage access to cloud resources.
 
     This class looks for several environment variables to configure
-    access to GCS resources. These can be set directly, or be in a
+    access to cloud resources. These can be set directly, or be in a
     .env file at the top level.
 
     The following variables need to be set:
 
-    GCS_BUCKET_NAME: Name of bucket where 10k filings are stored.
+    GCS_FILINGS_BUCKET_NAME: Name of bucket where 10k filings are stored.
+    GCS_LABELS_BUCKET_NAME: Name of bucket where labelled training data is stored.
```

Comment:
It might be good to specify here whether the `labeled` directory should be included in this bucket name, or if it should be the name of the bucket one up from that. Although, I guess the `labeled` directory isn't really the bucket, it's just a folder within the bucket.

Reply:
I made this a bit more specific that it should point to the top-level bucket. Let me know if this seems clear enough!

Reply:
Looks good!